test/queue/push_pop_queue.c

```diff
@@ -0,0 +1,46 @@
+#include <micro/util/queue.h>
+
+#include <assert.h>
+
+int main()
+{
+    queue_t *q = queue_new(10);
+
+    int el_1 = 23;
+    char el_2 = 'b';
+    float el_3 = 5.0556;
+
+    int resp;
```

Comment:
This is not the `response`, is the `error` this is the reason why we return 0 (the only value considered `false`) so we test: `if (error == 0)` or `if (!error)`